setup.cfg

```diff
@@ -0,0 +1,13 @@
+[check-manifest]
+ignore =
+    */test
+    */test/*
+    .*
+    node_modules
+
+[pep257]
+ignore = D202
+explain = true
+
+[yapf]
+based_on_style = pep8
```

Comment:
No no, leave it in, I didn't see the yapf stuff in h, if it's from h it can go in here.